db/migrate/20251116184506_add_user_id_to_places.rb

```
@@ -0,0 +1,12 @@
+class AddUserIdToPlaces < ActiveRecord::Migration[8.0]
+  disable_ddl_transaction!
+
+  def up
+    # Add nullable for backward compatibility, will enforce later via data migration
+    add_reference :places, :user, null: true, index: {algorithm: :concurrently} unless foreign_key_exists?(:places, :users)
```

RuboCop (1.81.7)

[convention] Space inside { missing.

(Layout/SpaceInsideHashLiteralBraces)